argo_workflows/assets/dashboards/argo_workflows.json

```diff
@@ -0,0 +1,756 @@
+{
+    "author_name": "Datadog",
+    "description": "[[suggested_dashboards]]",
+    "layout_type": "ordered",
+    "template_variables": [],
+    "title": "Argo Workflows",
+    "widgets": [
+        {
+            "definition": {
+                "banner_img": "/static/images/logos/argo-workflows_large.svg",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Argo Workflows",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "background_color": "white",
+                            "content": "**This dashboard provides observability for your  Argo Workflows deployments with the Argo Workflows Integration.**\n",
```

Comment:
```suggestion
                            "content": "**This dashboard provides observability for your Argo Workflows deployments with the Argo Workflows Integration.**\n",
```

nit

Reply:
it does look weird in the UI too, fixed with other stuff. thanks!